src/pages/Stepper.tsx

```diff
@@ -0,0 +1,31 @@
+import ModalProvider from '@/provider/modal-provider/ModalProvider';
+import useStepContext from '@/provider/step-provider/hook/useStepContext';
+import { AddCard, CardList, CardRegisterComplete } from '.';
+
+const Stepper = () => {
+  const { route } = useStepContext();
+
+  if (route === 'LIST') {
+    return <CardList />;
+  }
+
+  if (route === 'CARD') {
+    return (
+      <ModalProvider>
+        <AddCard />
+      </ModalProvider>
+    );
+  }
+
+  if (route === 'COMPLETE') {
+    return (
+      <ModalProvider>
+        <CardRegisterComplete />
+      </ModalProvider>
+    );
+  }
+
+  return <></>;
+};
```

Comment:
이 시점에서 이러한 코드를 깔끔하게 정리하는 좋은 코드 기법을 알려드리자면, 

```suggestion
const StepperContent = {
  LIST: <CardList />,
  CARD: (
    <ModalProvider>
      <AddCard />
    </ModalProvider>
  ),
  COMPLETE: (
    <ModalProvider>
      <CardRegisterComplete />
    </ModalProvider>
  ),
} as const;

const Stepper = () => {
  const { route } = useStepContext();
  const Component = StepperContent[route] ?? null;
  
  return Component;
};
```

이렇게 key-value 쌍으로 만들 수 있답니다!